odd-collector/odd_collector/adapters/postgresql/tests/conftest.py

```diff
@@ -0,0 +1,258 @@
+        is_self_referencing='NO', is_identity='NO', identity_generation=None, identity_start=None,
+        identity_increment=None, identity_maximum=None, identity_minimum=None, identity_cycle='NO',
+        is_generated='NEVER', generation_expression=None, is_updatable='YES', description=None, type_oid=23,
+        enums=[], is_primary=True
+    )
+
+
+@pytest.fixture(scope="module")
+def varchar_column():
+    return Column(
+        attrelid=16520, table_catalog='test', table_schema='test',
+        table_name='test', column_name='test', ordinal_position=2, column_default=None,
+        is_nullable='YES', data_type='varchar', character_maximum_length=255, character_octet_length=1020,
+        numeric_precision=None, numeric_precision_radix=None, numeric_scale=None, datetime_precision=None,
+        interval_type=None, interval_precision=None, character_set_catalog=None, character_set_schema=None,
+        character_set_name=None, collation_catalog=None, collation_schema=None, collation_name=None,
+        domain_catalog=None, domain_schema=None, domain_name=None, udt_catalog='test', udt_schema='pg_catalog',
+        udt_name='varchar', scope_catalog=None, scope_schema=None, scope_name=None, maximum_cardinality=None,
+        dtd_identifier='2', is_self_referencing='NO', is_identity='NO', identity_generation=None,
+        identity_start=None, identity_increment=None, identity_maximum=None, identity_minimum=None,
+        identity_cycle='NO', is_generated='NEVER', generation_expression=None, is_updatable='YES',
+        description=None, type_oid=1043, enums=[], is_primary=False
+    )
+
+
+@pytest.fixture(scope="module")
+def vector_column():
+    return Column(
+        attrelid=16520, table_catalog='test', table_schema='test',
+        table_name='test', column_name='test', ordinal_position=3, column_default=None,
+        is_nullable='YES', data_type='vector', character_maximum_length=None, character_octet_length=None,
+        numeric_precision=None, numeric_precision_radix=None, numeric_scale=None, datetime_precision=None,
+        interval_type=None, interval_precision=None, character_set_catalog=None, character_set_schema=None,
+        character_set_name=None, collation_catalog=None, collation_schema=None, collation_name=None,
+        domain_catalog=None, domain_schema=None, domain_name=None, udt_catalog='test', udt_schema='public',
+        udt_name='vector', scope_catalog=None, scope_schema=None, scope_name=None, maximum_cardinality=None,
+        dtd_identifier='4', is_self_referencing='NO', is_identity='NO', identity_generation=None,
+        identity_start=None, identity_increment=None, identity_maximum=None, identity_minimum=None,
+        identity_cycle='NO', is_generated='NEVER', generation_expression=None, is_updatable='YES',
+        description=None, type_oid=16420, enums=[], is_primary=False
+    )
+
+
+@pytest.fixture(scope="module")
+def table_without_vector_column(int4_id_column, varchar_column):
```

Comment:
Table without vector is basically just a table, we don't have to specify that there is no vector column in there
```suggestion
def table(int4_id_column, varchar_column):
```

Reply:
@AndreyNenashev done.